doc/nrf/glossary.rst

```diff
@@ -455,6 +485,13 @@ Glossary
    Operational Amplifier (op-amp)
       A high-gain voltage amplifier that has a differential input and, usually, a single output.
 
+   Orthogonal Frequency Division Multiplexing (OFDM)
+      A type of digital transmission and a method of encoding digital data on multiple carrier frequencies.
+
+   Orthogonal Frequency Division Multiple Access (OFDMA)
+      A multiple access mechanism for shared medium networks based on Orthogonal Frequency Division Multiplexing (OFDM) achieved by assigning subsets of channel sub-carriers to individual users.
```

Comment:
```suggestion
      A multiple access mechanism for shared medium networks based on :term:`Orthogonal Frequency Division Multiplexing (OFDM)` achieved by assigning subsets of channel sub-carriers to individual users.
```

Reply:
I can do this, but dunno whether it is necessary as the term is just above, next to this item.